backend/routes/incidents.py

```diff
@@ -1,13 +1,21 @@
-from flask import Blueprint
-from backend.database import Incidents
+from flask import Blueprint, Response, jsonify
+from backend.database import Incidents, IncidentSchema
 
 
 incident_routes = Blueprint(
     "incident_routes", __name__, url_prefix="/incidents"
 )
 
 
+@incident_routes.route("/test", methods=["GET"])
+def get_incidents():
+    incidents = Incidents.query.all()
+    incident_schema = IncidentSchema(many=True)
+    incidents_dump = incident_schema.dump(incidents)
+    return jsonify(incidents_dump)
+
+# define these routes last, otherwise they will override /<route> routes
 @incident_routes.route("/", methods=["POST", "GET"])
 @incident_routes.route("/<id>", methods=["PUT", "DELETE"])
 def get_single_incident(id=None):
-    return Incidents.get_delete_put_post(id)
+    return Incidents.get_delete_put_post(id)
```

Comment:
Whoops, must've done this by accident. Would it be worth setting up something like pylint to catch these in the future?

Reply:
We use flake8, which should catch it. You can run the command `flake8 backend/` to see those errors, and you can use the command `black backend/` to automatically prettify.

flake8 is also what we use in our continuous integration. That's why your PR has a red X instead of a green check:

<img width="1216" alt="image" src="https://user-images.githubusercontent.com/31971762/115475164-0eb5c480-a20d-11eb-97fa-77184f53bf7b.png">

Reply:
Running black and changing a test case from `GET /incidents/` to `GET /incidents` got me a green check! Seems like black might have edited some files I didn't though, I can revert the commit if need be.

Reply:
If black and flake8 say it's fine, then it's fine in my book.